src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
Nope, intending to use double equals here, as it's logically the same as `c.topic !== null && c.topic !== undefined`.  It's equivalent to `_.isNil`:

```
function isNil(value) {
    return value == null;
}
```